drivers/bridge/bridge.go

```diff
@@ -92,9 +92,14 @@ func init() {
 	portMapper = portmapper.New()
 }
 
-// New provides a new instance of bridge driver
-func New(dc driverapi.DriverCallback) (string, driverapi.Driver) {
-	return networkType, &driver{}
+// New constructs a new bridge driver
+func New() driverapi.Driver {
+	return &driver{}
+}
+
+// Init registers a new instance of bridge driver
+func Init(dc driverapi.DriverCallback) error {
```

Comment:
Do we really need a new function `Init()` and also a `New()` and both are exported ?
If the `New()` is kept just for convenience, then shall we unexport it ?
(even better, we can remove the `Init()` method and just have the `New()` or vice-versa.